package.json

```diff
@@ -11,10 +11,13 @@
   },
   "dependencies": {
     "axios": "^1.6.7",
+    "lodash": "^4.17.21",
```

Comment:
From what I can tell, we might not actually need `lodash` as a dependency here. Do you think we can remove it?